### PR TITLE
Create the items view in the subcategory scene

### LINKED ISSUE #29: Change the item & sub-category view
**Is your feature request related to a problem?**
No of steps from category to item is high. We need to minimize the number of clicks from category to item.

**Describe the solution you'd like**
Create a menu in the sub-category page & load the items in the sub-category page itself.

### PR DESCRIPTION
**Pull request by anjula-sack**
## Purpose

- The purpose of this PR is to fix #29 

## Goals

- Reduce steps to get to the items

## Approach

- Render the `items` component in the subcategory scene
Kapture 2020-06-25 at 21 27 51

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: app/src/scenes/SubCategories/styles.module.css [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
-.listItem{
-    padding: 20px;
+.menuItem{
     text-align: center;
+    margin-left: 30px;
+    margin-right: 30px;
+}
+
+.linkItem{
+    font-size: 20px;
```
Comment (jayasanka-sack) [inline app/src/scenes/SubCategories/styles.module.css:13]: You can move this to .menuItem

File: app/src/scenes/Items/index.tsx [hunk 2]
```diff
@@ -35,32 +31,8 @@ class Items extends React.Component<
 
   componentDidMount() {
     this.fetchItems();
-    this.fetchSubCategoryDetails();
```
Comment (jayasanka-sack) [inline app/src/scenes/Items/index.tsx:38]: You have to load it when the component gets mounted.

Comment (anjula-sack) [reply app/src/scenes/Items/index.tsx:38]: We use this to get the subcategory title but now we don’t show it.

Comment (jayasanka-sack) [reply app/src/scenes/Items/index.tsx:38]: okay!

**Pull request merged**